app/helpers/view_component_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ViewComponentHelper
+  # For each component, define an helper that renders the component
+  # Forward arguments, keywords and block
+  Dir[Rails.root.join("app/components/*.rb")].each { |file| require file }
```

Comment:
⚠️ How to ensure components are loaded here? Without these line, it works with ruby 2.7 + rails 7 but not with ruby 2.6.7 + rails 6.1.3.2... With these line it works but it does not seem to be the best way to do that...